cmd/constants.go

```diff
@@ -2,5 +2,4 @@ package cmd
 
 var (
 	defaultOutputFormat = "junitxml"
-	defaultLogFile      = "preflight.log"
```

Comment:
_see review-level comment_

This default should probably remain to remove hard-coded values elsewhere.